about.html

```diff
@@ -0,0 +1,154 @@
+    />
+    <link rel="stylesheet" href="css/about.css" />
+    <title>Haddis Technology!</title>
+  </head>
+  <body>
+    <div>
+      <div id="container">
+        <div class="nav-bar">
+          <nav class="primary-nav">
+            <ul class="primary-nav">
+              <li>
+                <a href="#"><i class="fab fa-facebook"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fab fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href="#" class="my-page"><p>MyPage</p></a>
+              </li>
+              <li>
+                <a href="#" class="logout log-pad"><p>Logout</p></a>
+              </li>
+            </ul>
+          </nav>
+          <nav class="secondary-nav">
+            <div class="banner secondary-nav">
+              <a href="index.html"
+                ><img
+                  class="logo-img"
+                  src="images/eagle-logo.jpeg"
+                  alt="logo"
+                />
+                <p class="comp-name">Haddis Tecnology</p></a
+              >
+            </div>
+            <ul class="secondary-nav nav-menu">
+              <li class="nav-item">
+                <a href="index.html" class="link-nav home-link">Home</a>
+              </li>
+              <li class="nav-item">
+                <a href="about.html" class="link-nav">About</a>
+              </li>
+              <li class="nav-item"><a href="" class="link-nav">Program</a></li>
+              <li class="nav-item"><a href="" class="link-nav">Join</a></li>
+              <li class="nav-item"><a href="" class="link-nav">Sponsor</a></li>
+              <li class="nav-item"><a href="" class="link-nav">News</a></li>
+              <li class="nav-item">
+                <a href="" class="confference">Confference</a>
+              </li>
+            </ul>
+          </nav>
+
+          <div class="hamburger" id="hamburger">
+            <span class="l-1 bar"></span>
+            <span class="l-2 bar"></span>
+            <span class="l-3 bar"></span>
+          </div>
+        </div>
+      </div>
+      <section class="headlines">
+        <div class="advert-txt">
+          <h1>Creative Idea Discustions 2022</h1>
+        </div>
+        <div class="box-txt">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure, ad
+            necessitatibus nesciunt voluptate numquam odit veniam reprehenderit
+            expedita doloremque laboriosam natus eligendi itaque, maxime
+            corporis dolorum ipsa neque deserunt. Voluptatem. Lorem ipsum dolor
+            sit amet consectetur adipisicing elit. Iure, ad necessitatibus
+            nesciunt voluptate numquam odit veniams
+          </p>
+        </div>
+        <div class="address-txt">
+          <p>
+            Please contact us per email for further quetsions about our campany!
+          </p>
+          <br />
+          <a href="#">kumilachew1212@gmail.com</a>
```

Comment:
- [ ] Please ensure you use the correct font-color for the email in the about page, right now it has the default styling as you can see below
![image](https://user-images.githubusercontent.com/90556221/181483726-cc0afe8c-0fae-4353-8ee5-c0f606799556.png)